src/ExpressionEvaluator/Core/Source/ExpressionCompiler/DkmUtilities.cs

```diff
@@ -43,19 +43,23 @@ private static IEnumerable<DkmClrModuleInstance> GetModulesInAppDomain(this DkmC
                 });
         }
 
-        internal unsafe static ImmutableArray<MetadataBlock> GetMetadataBlocks(this DkmClrRuntimeInstance runtime, DkmClrAppDomain appDomain)
+        internal static ImmutableArray<MetadataBlock> GetMetadataBlocks(
+            this DkmClrRuntimeInstance runtime, 
+            DkmClrAppDomain appDomain, 
+            ImmutableArray<MetadataBlock> previousMetadataBlocks = default(ImmutableArray<MetadataBlock>))
```

Comment:
Is the default value necessary? It looks like all callers are supplying the value.